.vscode/launch.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
this should be in git ignore?

Reply:
can be in gitignore if you wish so, most of the time I see it added so that people who use vscode already have the setup for it

Reply:
well it cant hurt I guess